pytest-backend/exasol/pytest_backend/__init__.py

```diff
@@ -33,6 +44,37 @@ def pytest_addoption(parser):
             but this is the same as the default.
             """,
     )
+    parser.addoption(
+        "--project-short-tag",
+        help="""Short tag aka. "abbreviation" for your current project.
+            See docstring in project_short_tag.py for more details.
+            pytest plugin for exasol-saas-api will include this short tag into
+            the names of created database instances.""",
+    )
```

Comment:
It would probably make more sense to name it in relation to the mechanism rather than based on what we are currently passing in.
E.g. `--db-prefix` or `--prefix`

Reply:
The idea (not in this ticket) was to have something identifying a project. It is currently used to form a temporary database name, but the thinking was wider than that.

Reply:
Understood, I think the project short tag can and maybe even should be mentioned in the help text. Having it as name for the parameter though can be misleading imho because the mechnasim to me seems more general and would work with "anything" (which is good). The current name would give the user a "False" sense of limitation in my opinion. Still If you want to keep it that's ok with me just wanted to bring it up.